### PR TITLE
Update cdk-stack.svg

### PR DESCRIPTION
**Pull request by mitchpolygon**
Update cdk fep image

### CODE REVIEW

**Review by Sharonbc01 — APPROVED** [review]
Looks great Mitch

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 1]
```diff
@@ -1,3 +1,236 @@
-!!! tip
-    The migration instructions for moving from fork 9 to fork 12 are coming soon.
+> 💡 For CDK fork ID9 chains **NOT attached to the AggLayer** (Isolated), go straight to section 5.
 
+> 💡 For CDK fork ID9 chains **attached to the AggLayer**, follow steps in sections 1 to 5. This is a coordinated effort between Polygon and the Implementation Provider.
+
+## 1. Summary of the Procedure
+
+To initiate a CDK chain upgrade, the Implementation Provider can request support from Polygon by submitting the "*Request Help for an Issue with an Existing CDK Chain*" through the service desk.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:7]: Link the words "service desk" to https://cdk.polygon.technology/

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 2]
```diff
@@ -1,3 +1,236 @@
...
+  <tr>
+    <td>
+      <img alt="CDK Service Desk" src="https://github.com/mitchpolygon/polygon-docs/blob/main/docs/img/cdk/CDK-service-desk.png?raw=true" width="300"/>
+    </td>
+    <td>
+      <img alt="Example Request" src="https://github.com/mitchpolygon/polygon-docs/blob/main/docs/img/cdk/Example-request.png?raw=true" width="300"/>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>CDK service desk</strong></td>
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:19]: I recommend better images that are both the same size and balanced on the page if possible. I know easy to ask for and hard to do. see service desk here to take screen shots https://cdk.polygon.technology/

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 3]
```diff
@@ -1,3 +1,236 @@
...
+
+# Example Maintenance Communication to Network Partners
+
+There is a planned maintenance window upgrade of the xxxx network on the following dates. This is to upgrade the xxx network from Fork ID9 to Fork ID12.
+
+**Maintenance Event:**  xxx Network Upgrade from Fork ID9 to Fork ID12
+
+**Date:** TBD by Implementation Provider
+
+**Time:** 00:00 PM EDT / 12:00 PM UTC
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:214]: replace 12:00 with 00:00

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 4]
```diff
@@ -1,3 +1,236 @@
...
+
+| **CDK Components**                         | **Fork ID 9**                                                                                                                                            | **CDK Components**                         | **Fork ID 12**                                                                                                            |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| CDK Validium node<br>Sequence sender<br>Aggregator | [0.6.7+cdk.1](https://hub.docker.com/layers/0xpolygon/cdk-validium-node/0.6.7-cdk.1/images/sha256-dafb15f9355331b4b7174f47ac416b275915ff24a9ed89c211c7c15c8adfc6b8?context=explore) | CDK Erigon RPC & CDK node                 | [cdk-erigon](https://github.com/0xPolygonHermez/cdk-erigon) 2.1.1 -- forkid.12 release *(Will link final version when released)* |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:47]: Replace with [cdk-erigon:v2.1.x](https://github.com/0xPolygonHermez/cdk-erigon/releases)

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 5]
```diff
@@ -1,3 +1,236 @@
...
+|                                            |                                                                                                                   | CDK node<br>Sequence sender<br>Aggregator | 0.3.0-rc2                                                                                                                 |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:48]: Replace with [cdk:v0.3.x](https://github.com/0xPolygon/cdk/releases)

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 6]
```diff
@@ -1,3 +1,236 @@
...
+| Tx pool manager                            | [zkevm-pool-manager](https://github.com/0xPolygon/zkevm-pool-manager)                                                                                    | Tx pool manager                            | [zkevm-pool-manager](https://github.com/0xPolygon/zkevm-pool-manager)                                                    |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:49]: add comment after link to repo  
Use latest tag

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 7]
```diff
@@ -1,3 +1,236 @@
...
+| Executor                                   |                                                                                                                   | Executor                                   | hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12                                                                            |
+| Prover                                     | [v6.0.0](https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v6.0.0)                                                                            | Prover                                     | hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12                                                                            |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:51]: Replace hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12 
 with [zkevm-prover](https://hub.docker.com/r/hermeznetwork/zkevm-prover/tags)  v8.0.0-RC14

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 8]
```diff
@@ -1,3 +1,236 @@
...
+|                                            |                                                                                                                   | CDK node<br>Sequence sender<br>Aggregator | 0.3.0-rc2                                                                                                                 |
+| Tx pool manager                            | [zkevm-pool-manager](https://github.com/0xPolygon/zkevm-pool-manager)                                                                                    | Tx pool manager                            | [zkevm-pool-manager](https://github.com/0xPolygon/zkevm-pool-manager)                                                    |
+| Executor                                   |                                                                                                                   | Executor                                   | hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12                                                                            |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:50]: We can remove this row as Executor is the same as prover.

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 9]
```diff
@@ -1,3 +1,236 @@
...
+| Prover                                     | [v6.0.0](https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v6.0.0)                                                                            | Prover                                     | hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12                                                                            |
+| CDK data availability                      | [v0.0.7](https://hub.docker.com/layers/0xpolygon/cdk-data-availability/0.0.7/images/sha256-17590789a831259d7a07d8a042ea87e381c5708dec3a7daef6f3f782f50b2c00?context=explore) | CDK data availability                      | [cdk-data-availability](https://github.com/0xPolygon/cdk-data-availability) v0.0.10                                       |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:52]: add 
Use latest tag

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 10]
```diff
@@ -1,3 +1,236 @@
...
+| zkEVM rollup node                          | [v6.0.0](https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v6.0.0)                                                                            | zkEVM rollup node                          | N/A                                                                                                                       |
+| Contracts                                  | [v6.0.0](https://github.com/0xPolygonHermez/zkevm-contracts/releases/tag/v6.0.0-rc.1-fork.9)                                                             | Contracts                                  | [zkevm-contracts](https://github.com/0xPolygonHermez/zkevm-contracts)                                                     |
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:54]: add
Use Fork ID12 contracts

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 11]
```diff
@@ -1,3 +1,236 @@
...
+| Bridge UI                                  | [Polygon Portal](https://portal.polygon.technology/)                                                                                                     | Bridge UI                                  | [Polygon Portal](https://portal.polygon.technology/)                                                                      |
+
+## 3. Implementation Provider Preparation Steps
+
+The Implementation Provider must prepare in advance for the upgrade to ensure a smooth transition from fork ID 9 to fork ID 12. Failure to complete these steps ahead of time could result in delays or even cancellation of the scheduled upgrade.
+
+1. The Implementation Provider downloads CDK Fork 12 components binaries/images in advance so they are ready to deploy.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:62]: Replace with CDK FEP Fork 12 components and link to https://docs.polygon.technology/cdk/releases/stack-components/#cdk-fep-components

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 12]
```diff
@@ -1,3 +1,236 @@
...
+    - Generate data stream files from the current `cdk-validium-node` database.
+        - Tool and instructions can be found here:  
+            - [https://github.com/0xPolygonHermez/zkevm-node/tree/develop/tools/datastreamer](https://github.com/0xPolygonHermez/zkevm-node/tree/develop/tools/datastreamer)
+    - Use the Erigon tool (from `cdk-erigon` repo) to serve the DS:
+        - `go run ./zk/debug_tools/datastream-host --file /path/to/directory`
+        - Start CDK-Erigon, which reads from this datastream (provide `zkevm.l2-datastreamer-url: 127.0.0.1:6900` in the config).
+    - Wait for it to sync to the tip.
+    - CDK-Erigon can be stopped. The generated files will be used later during the upgrade process.
+
+The whole process should look more orr less like this:
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:77]: replace with or

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 13]
```diff
@@ -1,3 +1,236 @@
...
+
+## 4. Polygon Preparation Steps
+
+1. Polygon will collaborate with the Implementation Provider to **schedule** the UTC timing and dates for the upgrade, incorporating required timelocks.
+2. Polygon will set up Google Meet calls between Polygon and the Implementation Provider's engineers to conduct planned upgrades for both testnet and mainnet on agreed dates.
+3. Polygon will prepare in advance and with agreed timelock:
+    - Rolluptype for fork 12
+    - Upgrade transaction to fork 12
+4. For chains attached to the Polygon Agglayer, Polygon will handle steps to upgrade the permissionless node.
+5. Polygon will share [example communication](https://docs.google.com/document/d/1zyXojlg4n2Th6P3y3Wqe8px15HL8FnBpSIWd54co1tE/edit) that Implementation Providers can use to prepare their customer network partners and communities.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:107]: This is a link to a google doc on Polygon shared drive so not fit for use here. I recommend you add this to the bottom of the page and have the link jump to that content there.

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 14]
```diff
@@ -1,3 +1,236 @@
...
+> 💡 For an **isolated chain not attached to the Polygon Agglayer**, the chain admin can perform operational step 6 on their chain’s rollup manager contract. Polygon is not involved.
+
+1. **Please Note:** Wait for Polygon to send the L1 transaction (tx) and confirm it.
+2. **Polygon:** Upgrade the Smart Contract (multisig):
+    - Upgrade rollup to fork 12.
+    - Wait for the Tx to be finalized.
+
+### Steps to Deploy CDK FEP Fork 12 Components
+
+1. [With the network stopped, repeat Erigon sync to get it fully synced to the current state.](https://www.notion.so/CDK-chain-upgrade-procedure-from-Fork-ID9-to-Fork-ID12-11980500116a802ab22cec6f7eea6080?pvs=21)
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:168]: This link can not be used as it link sto the Polygon notion page section. Please have the link bring the user to step 5 in the IP Preparation steps section.

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 15]
```diff
@@ -1,3 +1,236 @@
...
+7. Start the Bridge.
+8. Start the CDK aggregator and Sequence Sender components.
+9. Start the stateless Prover.
+
+### Polygon Steps for CDK Chains Attached to the Agglayer
+
+Polygon will be accountable for upgrading the Agglayer permissionless nodes during the upgrade process.
+
+1. When the Implementation Provider has stopped the sequencer.
+2. Polygon's v-team shuts down the CDK chain permissionless nodes hosted by Polygon:
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:190]: Replace with DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 16]
```diff
@@ -1,3 +1,236 @@
...
+    - Set `replicaCount` to 0 for `rpc`, `synchronizer`, and `executor` in the desired node definition as found [here](https://github.com/0xPolygon/helm-charts/tree/main/charts/permissionless-nodes/nodes).
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:191]: Remove this line as internal to Polygon DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 17]
```diff
@@ -1,3 +1,236 @@
...
+    - Run helm update to deploy changes, which will shut down the permissionless nodes.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:192]: Remove this line as internal to Polygon DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 18]
```diff
@@ -1,3 +1,236 @@
...
+3. Polygon v-team updates the Agglayer config:
+    - Change the `image.tag` to the desired version [here](https://github.com/0xPolygon/helm-charts/tree/main/charts/permissionless-nodes/nodes).
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:194]: Remove this line as internal to Polygon DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 19]
```diff
@@ -1,3 +1,236 @@
...
+4. When the Implementation Provider restarts the sequencer and everything is running as expected:
+5. Polygon starts the upgraded permissionless nodes to ID12:
+    - Set `replicaCount` to 3 for `rpc` and `executor`; set `replicaCount` to 1 for `synchronizer`.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:197]: Remove this line as internal to Polygon DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 20]
```diff
@@ -1,3 +1,236 @@
...
+    - Run helm update to deploy changes.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:198]: Remove this line as internal to Polygon DevOps team

File: docs/cdk/how-to/migrate/forkid-9-12.md [hunk 21]
```diff
@@ -1,3 +1,236 @@
...
+    - Monitor logs/service status in Datadog for progress.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/forkid-9-12.md:199]: Remove this line as internal to Polygon DevOps team

**Review by Sharonbc01 — COMMENTED** [review]
Hi Mitch had completed my review with a bunch of changes. Let me know if any questions.

File: docs/cdk/how-to/migrate/forkid-9-12.md (hunk 6 above)
Comment (mitchpolygon) [reply docs/cdk/how-to/migrate/forkid-9-12.md:49]: @Sharonbc01 what do you mean for this one?

**Pull request merged**